.github/.codecov.yml

```diff
@@ -1,3 +1,3 @@
 coverage:
-  round: up
+  round: down
```

Comment:
Maybe we need to change the precision? I think rounding down is the default behavior and we were seeing issues like what is happening in #1388 before we changed to rounding up.

Reply:
@freddyaboulton ah you are correct, this is the default behavior...

Reply:
And the same thing is happening in #1374 😂

Reply:
The codecov.io docs are super unclear about what "round" means:
> Which direction to you want to round the coverage value

They should have included an example... what's their algorithm for determining whether or not a job passes codecov? I think the algorithm is
* Compute coverage percentage before and after
* Take the signed difference, after - before
* Round that difference using the `round` policy, then truncate using `precision` digits after the decimal
* If the resulting number is less than 0, fail

But... I haven't found their docs which say whether or not this is true, hence further trial and error

Reply:
And in #1388 😅

Reply:
Oops [I spoke too soon](https://docs.codecov.io/docs/frequently-asked-questions#how-is-coverage-calculated):

```
round((hits / (hits + partials + misses)) * 100, 5) = Coverage Ratio
```
According to this, rounding up means a higher coverage ratio, which I think is what we want. So I'll decrease the precision but leave this set to "up". Does that make sense?